### PR TITLE
N8n 3194 output panels

### PR DESCRIPTION
**Pull request by mutdmour**

### CODE REVIEW

File: packages/editor-ui/src/store.ts [hunk 1]
```diff
@@ -470,6 +476,11 @@ export const store = new Vuex.Store({
 
 			state.stateIsDirty = true;
 			Vue.set(node, 'parameters', updateInformation.value);
+
+			if (!state.nodeMetadata[node.name]) {
+				Vue.set(state.nodeMetadata, node.name, {});
+			}
+			Vue.set(state.nodeMetadata[node.name], 'parametersLastUpdatedAt', new Date().getTime());
```
Comment (BHesseldieck) [inline packages/editor-ui/src/store.ts:483]: you can also use this for numeric timestamps 👍🏼 

```suggestion
			Vue.set(state.nodeMetadata[node.name], 'parametersLastUpdatedAt', Date.now());
```

**Review by BHesseldieck — APPROVED** [review]

File: packages/editor-ui/src/store.ts (hunk 1 above)
Comment (mutdmour) [reply packages/editor-ui/src/store.ts:483]: nice

**Comment by janober** [comment]
Thank you very much. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.172.0`

**Pull request merged**